docs/types.md

```diff
@@ -133,3 +133,43 @@ The following example applies 2em padding on top and bottom and 3em padding left
     "icon-padding": [2, 3]
 }
 ```
+
+## Projection Mode
+
+There are currently two projection modes implemented.
+
+- `mercator` - A web mercator projection
+- `globe` - A spherical projection
+
+
+**Use a single projection**
+
+```ts
+mode: "mercator"
```

Comment:
"mode" is not part of the style spec, right? I think this is a bit confusing, isn't it?

Reply:
I renamed `type` to `mode`, because `type` is notoriously overloaded in the style spec.

Reply:
My only concern about using `style.projection` is what will happen in the future if we decide to add things to projection settings, `mode` and `type` are synonymous, so I don't have strong feelings either way, but if you decide to change the style spec definition I think it should be noted somewhere besides this PR, right?

Reply:
okay, let's keep the nesting so we can expand projection settings down the line. I think it's best we do the rename to `mode`, because `type` is a reserved word, just like in typescript, due to it's role in type validation:

https://github.com/maplibre/maplibre-style-spec/blob/5788145317e1b7cf444a324207822706eec9071a/src/validate/validate.ts#L87-L102